sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Generated/Models/Request.Serialization.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.AI.AnomalyDetector.Models
+{
+#pragma warning disable AZC0012 // Avoid single word type names
+    public partial class Request : IUtf8JsonSerializable
```

Comment:
You won't be able to merge changed modified generated code. You can suppress this warning in a partial class.

Reply:
But this is a very very bad type name

Reply:
>You won't be able to merge changed modified generated code. You can suppress this warning in a partial class.

if we can't change the generated code then how can we add warning for [this](https://github.com/Azure/azure-sdk-for-net/pull/14293/files#diff-398ead792ec778734f59d4302a375d1f) enum?

Reply:
Ok, let's just suppress that globally and see what the plan with this library.

Reply:
And by globally I mean a NoWarn in the csproj

Reply:
>You won't be able to merge changed modified generated code. You can suppress this warning in a partial class.

if we can't change the generated code then how can we add warning for [this](https://github.com/Azure/azure-sdk-for-net/pull/14293/files#diff-398ead792ec778734f59d4302a375d1f) enum?

>But this is a very very bad type name

yeah, I agree